### PR TITLE
Change to Apache 2.0 license

### PR DESCRIPTION
**Pull request by mosabua**
We would approval on this from @regadas @elferherrera and @Amatrasan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ A [Trino](https://trino.io) client for [Node.js](https://nodejs.org/).
 
 [![@latest](https://img.shields.io/npm/v/trino-client.svg)](https://www.npmjs.com/package/trino-client)
 ![it-tests](https://github.com/regadas/trino-js-client/actions/workflows/it-tests.yml/badge.svg)
-![license](https://img.shields.io/github/license/regadas/trino-js-client)
+![license](https://img.shields.io/github/license/trinodb/trino-js-client)
```
Comment (mosabua) [inline README.md:7]: I assume that this will start working after merge.

File: package.json [hunk 2]
```diff
@@ -23,7 +23,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "packageManager": "yarn@3.2.1",
-  "license": "MIT",
+  "license": "Apache-2.0",
```
[the remark this answers is not in the document]
Comment (mosabua) [reply package.json:26]: Should be correct according to https://spdx.org/licenses/ and https://docs.npmjs.com/cli/v10/configuring-npm/package-json?v=true#license

**Review by regadas — APPROVED** [review]

**Comment by mosabua** [comment]
I will wait for @martint with merge .. and ideally also get approval from @elferherrera and @Amatrasan .. could you maybe contact them @regadas ?

**Comment by regadas** [comment]
@mosabua yeah I can try

**Review by elferherrera — APPROVED** [review]

File: LICENSE [hunk 3]
```diff
@@ -1,21 +1,204 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright
```
[the remark this answers is not in the document]
Comment (mosabua) [reply LICENSE:189]: It just has a placeholder...

[the remark this answers is not in the document]
Comment (mosabua) [reply LICENSE:189]: which is wrong and incomplete.. but it is what we do in the core Trino repo see https://github.com/trinodb/trino/blob/master/LICENSE#L190

[the remark this answers is not in the document]
Comment (mosabua) [reply LICENSE:189]: Otherwise the file is a copy of the file from the trino repo

**Comment by mosabua** [comment]
Thank you for approving @elferherrera 

Ideally you could also chime in @Amatrasan ...

**Comment by mosabua** [comment]
I think we are good to go here .. can you give a final approval @martint ?

File: LICENSE (hunk 3 above)
[the remark this answers is not in the document]
Comment (martint) [reply LICENSE:189]: In that case, just remove that part. The copyright is implied to be by each contributor or organization that authorized them to contribute such code.

[the remark this answers is not in the document]
Comment (mosabua) [reply LICENSE:189]: ok

[the remark this answers is not in the document]
Comment (mosabua) [reply LICENSE]: Done now.

**Review by martint — APPROVED** [review]

**Pull request merged**